internal/compiler/builtin_macros.rs

```diff
@@ -322,7 +322,12 @@ fn to_debug_string(
         Type::Float32 | Type::Int32 => expr.maybe_convert_to(Type::String, node, diag),
         Type::String => expr,
         // TODO
-        Type::Color | Type::Brush | Type::Image | Type::Easing | Type::Array(_) => {
+        Type::Color
+        | Type::Brush
+        | Type::Image
+        | Type::Easing
+        | Type::Array(_)
+        | Type::KeyboardShortcut => {
```

Comment:
I tired to return a String that contains the stringified definition, but that causes infinite recursion... I need to check out why that happens in detail.